src/mpvis/mpdfg/actions.py

```diff
@@ -138,6 +138,7 @@ def get_multi_perspective_dfg_string(
     cost_currency: str = "USD",
     rankdir: str = "TD",
     diagram_tool: str = "graphviz",
+    arc_thickness_by: str = "frequency",
```

Comment:
[REQ]
Use string literals.

```suggestion
    arc_thickness_by: Literal["frequency", "time"] = "frequency",
```